R/detect_biallelic_markers.R

```diff
@@ -114,7 +120,7 @@ detect_biallelic_markers <- function(data, verbose = FALSE) {
     biallelic <- max(biallelic$n)
     # if (length(biallelic) != 1) stop("Mix of bi- and multi-allelic markers is not supported")
 
-    if (biallelic > 4) {
```

Comment:
This was something that I wasn't quite sure about. I'm sure you had a good reason for setting this to 4, but it seems to me that this makes datasets with 3 or 4 alleles per locus considered biallelic. Is this intended?

Reply:
ha... good catch, this generated lots of discussion with colleagues. The reason was that you could find 4 allelic state with SNPs and lowering this threshold was causing problems with some biallelic VCF dataset.

Reply:
Thinking on how to best approach this. Because most of the bi-allelic markers will have only 2 not 4 so a prop threshold could be use to accommodate smaller dataset where using 4 cause a small biallelic dataset to be tagged multi-allelic or a very small multi-allelic dataset with say 3 alleles per markers to be tagged bi-allelic...

Reply:
Why is it that you distinguish biallelic and multiallelic datasets? Is it for performance reasons?

Reply:
sometimes it's faster, yes.
although the difference is only highlighted with huge datasets
and in some filtering steps.